library/extractor/src/main/java/com/google/android/exoplayer2/metadata/id3/TextInformationFrame.java

```diff
@@ -19,52 +19,77 @@
 
 import android.os.Parcel;
 import android.os.Parcelable;
+import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import com.google.android.exoplayer2.MediaMetadata;
 import com.google.android.exoplayer2.util.Util;
+import com.google.common.collect.ImmutableList;
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 
 /** Text information ID3 frame. */
 public final class TextInformationFrame extends Id3Frame {
-
   @Nullable public final String description;
+
+  /** @deprecated Use {@code values[0]} instead. */
+  @Deprecated
   public final String value;
 
-  public TextInformationFrame(String id, @Nullable String description, String value) {
+  @NonNull
+  public final List<String> values;
```

Comment:
Can you add javadoc guaranteeing that this will always have at least one element?

Something like:

> The values of this ID3 frame. Will always have at least one element.